explorer-ui/src/components/MatchGrid/MatchGrid.jsx

```diff
@@ -39,6 +39,10 @@ export default function MatchGrid({ profiles, userProfile }) {
     }
   };
 
+  // Rank Matches
+  const rankMatches = (matches) =>{
+    let highestOverlap = 0;
```

Comment:
no return ?

Reply:
No, i'm setting a state based off an operation performed in this function

Reply:
Is that an incorrect way of doing things?